tests/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 add_subdirectory(reals)
 
 if(${run_unittests})
-
+    build_test_folder(flags_to_string_ut)

```

Comment:
Consider adding this in sorted order. Just so that it looks nice :)

Reply:
considered and not done.